crates/matrix-sdk-crypto/src/identities/manager.rs

```diff
@@ -2213,43 +2208,30 @@ pub(crate) mod tests {
         let own_identity =
             machine.get_identity(DataSet::own_id(), None).await.unwrap().unwrap().own().unwrap();
```

Comment:
This line looks similar: can it be simplified too?

Reply:
no. We need to call `own_identity.is_identity_signed()`; `is_identity_signed` is specific to `OwnUserIdentity`.